package.json

```diff
@@ -189,6 +195,7 @@
     "eslint-plugin-jsx-a11y": "2.2.3",
     "eslint-plugin-mocha": "4.8.0",
     "eslint-plugin-react": "6.8.0",
+    "identity-obj-proxy": "3.0.0",
```

Comment:
This too

Reply:
I need this for mocking svg file import in the tests

Reply:
we should probably use `babel-plugin-transform-require-ignore`  we have an issue raised to remove some legacy isomorphic code.

Reply:
You can do it natively with jest, see: https://github.com/quran/quran.com-frontend/pull/962/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R42

Reply:
okay @mmahalwy will copy it into the PR

Reply:
Done!.